lib/ui/shared/zuri_appbar.dart

```diff
@@ -29,6 +30,7 @@ class ZuriAppBar extends StatelessWidget implements PreferredSizeWidget {
     this.title,
     this.leadingWidth = false,
     this.subtitle,
+    this.leadingColor,
```

Comment:
This leading color will affect other parts of the app since there is no default value to it.

And besides the leading color you indicated later in the app is not in the video you recorded.